namespaces/nsenter/nsenter.c

```diff
@@ -151,6 +151,19 @@ void nsenter()
 			exit(1);
 		}
 	}
+	// read continue signal from fd 3. the signal is 1
+	// this is so that the parent can put the proces inside correct cgroups
```

Comment:
haha yes !. How about `blocking until the parent placed the process inside correct cgroups.`. I think that's sufficient.